pcdsdevices/pim.py

```diff
@@ -88,11 +88,11 @@ class PIM(BaseInterface, GroupDevice):
     _prefix_start = ''
 
     state = Cpt(PIMY, '', kind='normal')
-    zoom_motor = FCpt(IMS, '{self._prefix_zoom}', kind='normal')
+    zoom = FCpt(IMS, '{self._prefix_zoom}', kind='normal')
```

Comment:
I don't mind these shortenings. We likely put the longer versions in out of an abundance of caution not to collide with other things, but it seems like that isn't an issue.